general-superstaq/general_superstaq/finance.py

```diff
@@ -36,7 +37,8 @@ def read_json_minvol(json_dict: gss.MinVolJson) -> MinVolOutput:
 
 
 @dataclass
-class MaxSharpeOutput:  # pylint: disable=missing-class-docstring
+class MaxSharpeOutput:
+    """Data type for Maximum Volume Model output with Sharpe Ratio"""
```

Comment:
nit 
```suggestion
    """A class to store data from the Max Sharpe Ratio endpoint"""
```

Reply:
I also think there is a typo (not from your PR) in the docstring of `read_json_maxsharpe`. I think it should probably read: "Reads out returned JSON from SuperstaQ API's maxsharpe endpoint."